features/facebookImport/src/model/analyses/about-pictures-data-analysis.js

```diff
@@ -0,0 +1,62 @@
+import {
+    relevantZipEntries,
+    removeEntryPrefix,
+} from "../../importer/importer-util";
+import RootAnalysis from "./root-analysis";
+
+/**
+ * Minimum number of picutures that should be present in
+ * the export in order for this analysis to be active.
+ */
+const PICTURES_THRESHOLD = 3;
+
+export default class AboutPicturesDataAnalysis extends RootAnalysis {
+    get label() {
+        return RootAnalysis.Labels.TECH_DEMO;
+    }
+
+    get title() {
+        return "About Pictures Story";
+    }
+
+    /**
+     * Determine the pictures posted by the user.
+     *
+     * At the moment, since we do not import posts and albums,
+     * we count only the number of picture files in the folder
+     * "photos_and_videos" where posted pictures are exported.
+     *
+     * Depending on the export that folder might contain also
+     * other picture files like "emptyalbum.png" which was not
+     * uploaded by a user, but most likely added by Facebook.
+     *
+     * Also some folders like "thumbnails" indicate that pictures
+     * there are created from other pictures.
+     *
+     * For now we do not try to distinguish them from pictures
+     * uploaded by a user, or do any other classification. We
+     * simply include all pictures in that folder.
+     */
+    async _userPicturesFromExport(zipFile) {
+        const relevantEntries = await relevantZipEntries(zipFile);
+        return relevantEntries.filter((zipEntry) =>
+            /^photos_and_videos\/(.+)\.(png|jpg|jpeg|gif)$/.test(
```

Comment:
I see several possible issues here:
* PNGs and GIFs are rather used for memes, not for pictures-pictures
* I thought that the threshold included profile pictures, but it looks like it does not. In that case it should probably be 1
* Checking the extension would probably be enough. Even if pictures are not in that folder, they are likely to be analyzed by Facebook.

Reply:
The decision from the product size is to explicitly exclude messages and only take into account pictures uploaded in albums and posts. We could probably exclude GIFs, but are you sure that we should also exclude pngs?

Reply:
PNGs are mainly memes, badges and things like that. It's very unlikely a personal picture of the nature we're talking about here will be in PNG format. Besides, we get rid of things like emptyalbum.png

Reply:
`emptyalbum.png` is just what I had in my export. Maybe others will have `emptyalbum.jpg` or something completely different. Thumbnails for example are also jpg. But we could just select jpegs. Based on our current understanding of the export it should be ok.